openAPI/flight-board.yaml

```
@@ -0,0 +1,181 @@
+              $ref: '#/components/schemas/TicketPurchaseInput'
+      responses:
+        '200':
+          description: Успешная передача информации
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TicketPurchaseResponse'
+              examples:
+                ValidRequest:
+                  value:
+                    status: "success"
+        '400':
+          description: Ошибка в запросе
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                MissingFlightNumber:
+                  value:
+                    errorCode: 100
+                    message: "FlightNumber is required"
+        '500':
+          description: Внутренняя ошибка сервера
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                InternalServerError:
+                  value:
+                    errorCode: 500
+                    message: "InternalServerError"
+
+components:
+  schemas:
+    FlightRegistrationInput:
+      type: object
+      properties:
+        flightNumber:
+          type: string
+          description: Номер рейса для открытия регистрации.
+      required:
+        - flightNumber
+
+    FlightRegistrationResponse:
+      type: object
+      properties:
+        status:
+          type: string
+          description: Статус выполнения операции.
+      required:
+        - status
+
+    TicketPurchaseInput:
+      type: object
+      properties:
+        flightNumber:
+          type: string
+          description: Номер рейса, на который открыта продажа билетов.
+      required:
+        - flightNumber
+
+    TicketPurchaseResponse:
+      type: object
+      properties:
+        status:
+          type: string
+          description: Статус выполнения операции.
+      required:
+        - status
+
+    FlightStartInput:
+      type: object
+      properties:
+        flightNumber:
+          type: string
+          description: Номер рейса, уведомляющий о его начале.
+        startTime:
+          type: string
+          format: date-time
+          description: Время начала рейса.
+      required:
+        - flightNumber
+        - startTime
+
+    SuccessResponse:
+      type: object
+      properties:
+        status:
+          type: string
+          description: Статус выполнения действия.
+        message:
+          type: string
+          description: Сообщение, подтверждающее успешное выполнение действия.
+      required:
+        - status
+        - message
+
+    ErrorResponse:
+      type: object
+      properties:
+        errorCode:
+          type: integer
+          description: Код ошибки.
+        message:
+          type: string
+          description: Сообщение об ошибке.
+      required:
+        - errorCode
```

Нету схемы, на которую ссылается этот реф
```suggestion
                $ref: '#/components/schemas/ErrorResponse' 
```